tests/ci/cdk/cdk/accp_github_fuzz_ci_stack.py

```diff
@@ -0,0 +1,131 @@
+                        assumed_by=iam.ServicePrincipal("codebuild.amazonaws.com"),
+                        inline_policies=inline_policies)
+
+        # Create the VPC for EFS and CodeBuild
+        public_subnet = ec2.SubnetConfiguration(name="PublicFuzzingSubnet", subnet_type=ec2.SubnetType.PUBLIC)
+        private_subnet = ec2.SubnetConfiguration(name="PrivateFuzzingSubnet", subnet_type=ec2.SubnetType.PRIVATE_WITH_EGRESS)
+
+        # Create a VPC with a single public and private subnet in a single AZ. This is to avoid the elastic IP limit
+        # being used up by a bunch of idle NAT gateways
+        fuzz_vpc = ec2.Vpc(
+            scope=self,
+            id="{}-FuzzingVPC".format(id),
+            subnet_configuration=[public_subnet, private_subnet],
+            max_azs=1
+        )
+        build_security_group = ec2.SecurityGroup(
+            scope=self,
+            id="{}-FuzzingSecurityGroup".format(id),
+            vpc=fuzz_vpc
+        )
+
+        build_security_group.add_ingress_rule(
+            peer=build_security_group,
+            connection=ec2.Port.all_traffic(),
+            description="Allow all traffic inside security group"
+        )
+
+        efs_subnet_selection = ec2.SubnetSelection(subnet_type=ec2.SubnetType.PRIVATE_WITH_EGRESS)
+
+        # Create the EFS to store the corpus and logs. EFS allows new filesystems to burst to 100 MB/s for the first 2
+        # TB of data read/written, after that the rate is limited based on the size of the filesystem. As of late
+        # 2021 our corpus is less than one GB which results in EFS limiting all reads and writes to the minimum 1 MB/s.
+        # To have the fuzzing be able to finish in a reasonable amount of time use the Provisioned capacity option.
+        # For now this uses 100 MB/s which matches the performance used for 2021. Looking at EFS metrics in late 2021
+        # during fuzz runs EFS sees 4-22 MB/s of transfers thus 100 MB/s gives lots of buffer and allows ~4-5 fuzz runs
+        # to start at the same time with no issue.
+        # https://docs.aws.amazon.com/efs/latest/ug/performance.html
+        fuzz_filesystem = efs.FileSystem(
+            scope=self,
+            id="{}-FuzzingEFS".format(id),
+            file_system_name="AWS-LC-Fuzz-Corpus",
+            enable_automatic_backups=True,
+            encrypted=True,
+            security_group=build_security_group,
+            vpc=fuzz_vpc,
+            vpc_subnets=efs_subnet_selection,
+            performance_mode=efs.PerformanceMode.GENERAL_PURPOSE,
+            throughput_mode=efs.ThroughputMode.PROVISIONED,
+            provisioned_throughput_per_second=Size.mebibytes(100),
+        )
+
+        # Define CodeBuild.
+        fuzz_codebuild = codebuild.Project(
+            scope=self,
+            id="FuzzingCodeBuild",
+            project_name=id,
+            source=git_hub_source,
+            role=role,
+            timeout=Duration.minutes(120),
```

Comment:
note for later PR: should we tighten this to be closer to our expected running time?

Reply:
Maybe, but I did just find it useful for doing a one-off "long build" to see if we find anything interesting. The slack gives us more flexibility in dialing up/down the run-time which I like.